AppKit/CPTextView/CPTextView.j

```diff
@@ -1848,18 +1848,18 @@ var kDelegateRespondsTo_textShouldBeginEditing
 
 + (JSObject)_wordBoundaryRegex
 {
-    return /(^[0-9][\.,])|(^.[^-\.,+#'"!§$%&/\(<\[\]>\)=?`´*\s{}\|¶])/m;
+    return new RegExp("(^[0-9][\\.,])|(^.[^-\\.,+#'\\\"!§$%&/\\(<\\[\\]>\\)=?`´*\\s{}\\|¶])", "m");
```

Comment:
You are right. The triple backslashes are mistaken. No idea why I did this.
A single backslash is needed to escape the doublequote.